app/1.0/start/toolbox/add-elements.md

```diff
@@ -8,79 +8,67 @@ subtitle: "Build an app with App Toolbox"
 Now that you've added a new view to your application, you can start building
 out the details of that view.
 
-In the process, you'll likely want to turn
-to some off-the-shelf components, for example from the
-[Polymer Element Catalog][catalog] or community catalogs like
-[customelements.io][ceio].
+One of the appeals of the web component approach to web development 
+is the ecosystem. Rather than building everything in your app from scratch,
+you can find and use off-the-shelf elements that suit your needs. Two examples
+of places you can find reusable components are the 
+[Polymer Element Catalog][catalog] (elements built by the Polymer team) and 
+[customelements.io][ceio] (elements built by the web components community). 
 
-## Ensure bower is installed
+In this tutorial you install a third-party component and use it in the new
+page that you created in the last step. The element that you're going to 
+install is called `paper-slider`. It's a simple UI subcomponent that lets you 
+drag a slider bar. 
 
-[Bower][bower] is a front-end package manager which is the most common
-tool used for fetching and managing web components.
+## Install a third-party component
 
-Ensure it is installed by running the following command:
+Usually, when working with third-party components, you'd need to browse the
+catalogs and find one that suits your needs, and then get the command for
+installing the element from the element's page on the catalog. For example,
+you can see the command for installing `paper-slider ` on the left-hand side
+of its [documentation page][paper-slider]. This tutorial is just going to 
+give you the command directly.
 
-    npm install -g bower
+1. Run this command from your project root directory to install your new
+   element. 
 
-## Install a 3rd-party component
+       bower install --save PolymerElements/paper-slider
 
-Once you've identified a component you'd like to install, you'll want to find
-the bower package name for the component.
+The `--save` flag instructs Bower to update `bower.json` to include this 
```

Comment:
An alert/note like this should start with a bolded, run-in heading, LIke:

```
**Look up in the sky!** It's a bird... it's a plane... It's... wait, no, it's a bird.
{.alert .alert-info}
```

In this case, I think it would make as much sense to just drop the {.alert .alert-info} styling and make this a regular paragraph.